components/brave_wallet_ui/common/hardware/types.ts

```diff
@@ -67,3 +68,5 @@ export const FilecoinNetworkLocaleMapping = {
   [BraveWallet.FILECOIN_MAINNET]: 'Filecoin Mainnet',
   [BraveWallet.FILECOIN_TESTNET]: 'Filecoin Testnet'
 }
+
+export const DerivationBatch = 4
```

Comment:
nit: is this name descriptive enough - I don't really understand what it is

Reply:
done